packages/valory/skills/abstract_round_abci/__init__.py

```diff
@@ -20,6 +20,11 @@
 """This module contains an abstract round ABCI skill template for an AEA."""
 
 from aea.configurations.base import PublicId
+from hypothesis import settings
 
 
 PUBLIC_ID = PublicId.from_str("valory/abstract_round_abci:0.1.0")
+CI = "CI"
+
+
+settings.register_profile(CI, deadline=5000)
```

Comment:
We need a larger deadline on CI so it does not report flaky tests.

Reply:
how does this recognize it's running on CI? on open-aea I introduced an env var for this in the [workflow.yaml](https://github.com/valory-xyz/open-aea/blob/ca2140f3bd21bc1cdbf20dad8f33d727e982bfff/.github/workflows/workflow.yml#L10-L11)

Reply:
GH [sets it automatically](https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables). This should do the job, no?

Reply:
very nice, I didn't know